homeassistant/components/emoncms/config_flow.py

```diff
@@ -94,6 +94,12 @@ async def async_step_user(
                 self.include_only_feeds = user_input.get(CONF_ONLY_INCLUDE_FEEDID)
                 self.url = user_input[CONF_URL]
                 self.api_key = user_input[CONF_API_KEY]
+                emoncms_client = EmoncmsClient(
+                    self.url, self.api_key, session=async_get_clientsession(self.hass)
+                )
+                emoncms_unique_id = await emoncms_client.async_get_uuid()
+                await self.async_set_unique_id(unique_id=emoncms_unique_id)
```

Comment:
Avoid single-use variable, and there is no need to use a keywork argument for the unique_id.
```suggestion
                await self.async_set_unique_id(await emoncms_client.async_get_uuid())
```